addons/survey/security/survey_security.xml

```diff
@@ -35,24 +35,26 @@
             <field name="perm_read" eval="1"/>
             <field name="perm_create" eval="1"/>
         </record>
-        <record id="survey_survey_rule_survey_user_read" model="ir.rule">
-            <field name="name">Survey: officer: read all</field>
+
+        <record id="survey_survey_rule_survey_user_rw" model="ir.rule">
+            <field name="name">Survey: officer: read/write survey without responsible</field>
             <field name="model_id" ref="survey.model_survey_survey"/>
-            <field name="domain_force">[(1, '=', 1)]</field>
+            <field name="domain_force">[('user_ids', '=', False)]</field>
             <field name="groups" eval="[(4, ref('group_survey_user'))]"/>
             <field name="perm_unlink" eval="0"/>
-            <field name="perm_write" eval="0"/>
+            <field name="perm_write" eval="1"/>
             <field name="perm_read" eval="1"/>
             <field name="perm_create" eval="0"/>
         </record>
-        <record id="survey_survey_rule_survey_user_cwu" model="ir.rule">
-            <field name="name">Survey: officer: create/write/unlink own only</field>
+        
+        <record id="survey_survey_rule_survey_user" model="ir.rule">
+            <field name="name">Survey: officer: survey responsible only</field>
```

Comment:
I feel like those 2 access rules could be merged, you can write anyway in the first access rules so you can remove all fields

Also if we are the creator, we should always be allowed to read / write (otherwise you create a survey, you add someone else reponsibible, you save => access error)

we will have something like
```
['|', '|', ('user_ids', '=', False), ('user_ids', 'in', user.id), ('create_uid', '=', user.id)]
```

What do you think ?